src/main/osd/osd.c

```diff
@@ -545,6 +548,18 @@ void osdInit(displayPort_t *osdDisplayPortToUse, osdDisplayPortDevice_e displayP
     }
 }
 
+#ifdef USE_GPS_LAP_TIMER
+void printLapTime(char* buffer, uint32_t time) {
+    if (time != 0) {
+        const int timeSeconds = time / 1000;
+        const int timeDecimals = ((time % 1000) + 5) / 10;
```

Comment:
I think we should round to the nearest centisecond also for `timeSeconds` so that both variables are consistently rounded.
```suggestion
void printLapTime(char* buffer, uint32_t timeMs) {
    if (timeMs != 0) {
        timeMs += 5;  // round to nearest centisecond (+/- 5ms)
        const int timeSeconds = timeMs / 1000;
        const int timeDecimals = (timeMs % 1000) / 10;
```